.github/workflows/slack_notify_sql_parser.yml

```diff
@@ -0,0 +1,85 @@
+# Copyright 2020 The Actions Ecosystem Authors
+# Modifications Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Portions of this file are derived from the README examples in the Action
+# Slack Notifier project. The original source code was retrieved on
+# January 5, 2022 from:
+#
+#     https://github.com/actions-ecosystem/action-slack-notifier/blob/fc778468d09c43a6f4d1b8cccaca59766656996a/README.md
+
+# Send a notification to the #epd-sql-council Slack channel when a change
+# to the SQL parser is made.
+#
+# A notification is sent when all of these conditions are true:
+#   * A ready-to-review PR is (re-)opened, or a PR is moved from draft
+#     to ready-to-review.
+#   * The PR modifies the 'src/sql-parser/src/parser.rs' file.
+
+name: Slack SQL Parser Notifications
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - reopened
+      - ready_for_review
+    paths:
+      - "src/sql-parser/src/parser.rs"
+
+jobs:
+  notify:
+    name: "Notify about changes to the SQL parser"
+    runs-on: ubuntu-latest
+    if: ${{ !github.event.pull_request.draft }}
+    steps:
+      - name: "Path filter"
+        id: filter
+        uses: dorny/paths-filter@v2
+        with:
+          filters: |
+            parser-change:
+              - 'src/sql-parser/src/parser.rs'
```

Comment:
Can you do everything in src/sql-parser and src/sql-lexer? It's unlikely that you can make a change to the parser without touching parser.rs, but it _is_ possible and I'd rather err on the safe side for now.